src/components/EventCard.tsx

```diff
@@ -16,87 +19,107 @@ export type EventCardProps = {
+      position="relative"
+      marginTop={{ base: "0", md: 0 }}
+      _before={clearStyles}
+      _after={clearStyles}
+      width={{ base: "100%", md: "100%", xl: "100%" }}
+      height={"100%"}
     >
-      <Text color="primary.base" marginBottom={0} textAlign="end">
-        {date}
-        <Emoji text=":spiral_calendar:" fontSize="md" ms={2} />
-      </Text>
-      <Text marginBottom={0} textAlign="end">
-        <Text as="span" opacity={0.6}>
-          {location}
-        </Text>
-        <Emoji text=":round_pushpin:" fontSize="md" ms={2} />
-      </Text>
-      <Heading as="h3" marginTop={0} fontWeight="semibold" lineHeight={1.4}>
-        {title}
-      </Heading>
-      <Text>{description}</Text>
-      <ButtonLink href={to}>View Event</ButtonLink>
+      <Flex
+        borderRadius="sm"
+        border="1px solid"
+        borderColor="lightBorder"
+        height={"100%"}
+        direction={"column"}
+        justifyContent={"space-between"}
+        rounded={"4px"}
+      >
+        <Box>
+          <Flex
+            alignItems={"center"}
+            justifyContent={"center"}
+            background={"grayBackground"}
+            padding={2}
+            gap={2}
+            borderBottom="1px solid"
+            borderColor="primary.base"
+            roundedTop={"4px"}
+          >
+            <Icon as={BsCalendar3} boxSize={6} color="primary.base" />
+
+            <Text color="primary.base" marginBottom={0} textAlign="end">
+              {formatedDate}
+            </Text>
+          </Flex>
+
+          {/* TODO : add image hostname to next config or add event image to public dir  */}
+          <Flex
+            alignItems="center"
+            justifyContent="center"
+            boxShadow="rgb(0 0 0 / 10%) 0px -1px 0px inset;"
+          >
+            <Image
+              src={imageUrl ? imageUrl : "/images/events/event-placeholder.png"}
+              alt={title}
+              width={{ base: "100%", sm: "100%" }}
+              height={{ base: "224px", xl: "124px" }}
+              objectFit={"cover"}
+              fallbackSrc="/images/events/event-placeholder.png"
+            />
+          </Flex>
+          <Box padding={4}>
+            <Box textAlign={"center"}>
+              <Heading
+                as="h3"
+                fontSize={{ base: "md", md: "2xl" }}
+                marginTop={0}
+                fontWeight="semibold"
+                lineHeight={1.4}
+              >
+                {title}
+              </Heading>
+              <Text as="span" opacity={0.6}>
+                {location}
+              </Text>
+            </Box>
+            <Box>
+              <Text fontSize={{ base: "sm", md: "sm" }} mb={0} noOfLines={4}>
+                {description}
+              </Text>
+            </Box>
+          </Box>
+        </Box>
+        <Box padding={4} paddingTop={0} width={"100%"}>
+          <ButtonLink href={to} width={"100%"} variant="outline">
+            View Event
```

Comment:
Noting as a separate task we should extract this string for translation.

Reply:
Should I add it to `en` translations?